features/step_definitions/work_steps.rb

```diff
@@ -663,3 +680,9 @@
 Then /^the Remove Me As Chapter Co-Creator option should not be on the ([\d]+)(?:st|nd|rd|th) chapter$/ do |chapter_number|
   step %{I should not see "Remove Me As Chapter Co-Creator" within "ul#sortable_chapter_list > li:nth-of-type(#{chapter_number})"}
 end
+
+Then /^I should find myself on Chapter ([\d]+) of the work "([^\"]*)"$/ do |chapter_number, work_title|
```

Comment:
You can use `(\d+)` instead of `([\d]+)`, `(.*?)` instead of `([^\"]*)`.

Reply:
I changed to `(.*?)` in #2779.